packages/database/src/schema.prisma

```diff
@@ -159,171 +158,124 @@ model AppealToUserRole {
 /// Cases aren't linked to users directly, but there's a 1:1 mapping to Appeal, which has a link to user
 // NOTE: field names match the model rather than the usual camelCase. This avoids extra mapping.
 model AppealCase {
-  /// this ID is for front office only and won't correlate to other systems
-  id String @id @default(dbgenerated("newid()")) @db.UniqueIdentifier
-
   //***************************************************************************
-  // general fields
+  // ids/refs
   //***************************************************************************
 
-  /// unique case reference
-  caseReference String @unique
-  LPACode       String
+  id            String @id @default(dbgenerated("newid()")) @db.UniqueIdentifier /// this ID is for front office only and won't correlate to other systems
+  caseReference String @unique /// External case identifier
+  caseId        Int? /// Internal case identifier
 
   //***************************************************************************
-  // appellant case fields
+  // general case fields
   //***************************************************************************
 
-  LPAName                         String
-  appealTypeCode                  String // type shortcode such as HAS or S78
-  appealTypeName                  String // type name such as Householder or Full Planning
-  decision                        String
-  originalCaseDecisionDate        DateTime
-  costsAppliedForIndicator        Boolean
-  LPAApplicationReference         String
-  appellantCasePublished          Boolean  @default(false)
-  appellantProofEvidenceSubmitted Boolean  @default(false)
-  appellantProofEvidencePublished Boolean  @default(false)
-  appellantFinalCommentsSubmitted Boolean  @default(false)
-
-  appellantFirstName String @default("")
-  appellantLastName  String @default("")
-
-  siteAddressLine1    String
-  siteAddressLine2    String?
-  siteAddressTown     String?
-  siteAddressCounty   String?
-  siteAddressPostcode String
+  LPACode        String /// A unique identifier for the Local Planning Authority
+  appealTypeCode String? /// The type shortcode for an appeal type, e.g. HAS (Householder)
+  CaseType       CaseType?      @relation(fields: [appealTypeCode], references: [processCode])
+  caseStatus     String /// The processing status for the appeal
+  caseProcedure  String? /// The type of procedure for the appeal
+  ProcedureType  ProcedureType? @relation(fields: [caseProcedure], references: [key])
```

Comment:
type, status, and procedure: should we use tables for these or not?

lpa can do later as we need to move over from mongo

Reply:
have done all but the yes no some fields...